guides/echo-guides/echo-mjml.mdx

```diff
@@ -0,0 +1,315 @@
+---
+title: 'How to send notifications with Next.js and MJML'
+description: 'Leverage MJML package to send email notifications'
+---
+
+# Introduction
+
+This guide will walk you through how to send notifications with MJML and Novu. You can check out the code for a [sample demo app](https://github.com/sumitsaurabh927/echo-mjml).
```

Comment:
the repo URL should be https://github.com/novuhq/echo-mjml. You can transfer the repo to me so I transfer to Novu @sumitsaurabh927